sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -490,6 +440,9 @@ the client library to use the Netty HTTP client. Configuring or changing the HTT
 The following section provides several code snippets illustrating common patterns used in the Form Recognizer API.
 These code samples show common scenario operations with the Azure Form Recognizer client library.
 
+* Recognize business card from a URL: [RecognizeBusinessCardFromUrl][recognize_business_cards_from_url]
+* Recognize identity documents from a URL: [RecognizeIdentityDocumentsFromUrl][recognize_identity_documents_from_url]
+* Recognize invoice from a URL: [RecognizeInvoiceFromUrl][recognize_invoices_from_url]
```

Comment:
looks like for receipt and other samples, there is an option of file and url. could you add the sample links for files here too?

Reply:
there are some samples are missing. I will be address it in the sample issue #21139